app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.jsx

```diff
@@ -82,12 +96,43 @@ export class InputConditionalAttributes extends Component {
     }
   };
 
+  parseQueryAttributes = (value) => {
+    if (!value.value) return [];
+    const attributes = value.value.split(',').map((item) => {
+      if (item.includes(':')) {
+        if (item.indexOf(':') === item.length - 1) {
+          return {
+            key: item.slice(0, -1),
+            value: '',
+          };
+        } else {
+          const values = item.split(':');
+          return {
+            key: `${values[0]}`,
+            value: values[1],
+          };
+        }
+      } else {
+        return {
+          key: '',
+          value: item,
+        };
+      }
+    });
+    return attributes;
+  };
+
   onChangeTags = (tags) => {
-    const newAttributes = [...this.state.attributes, { key: tags.key, value: tags.value }];
+    const newAttributes = [
+      ...this.state.attributes,
+      { key: tags.key || '', value: tags.value || '' },
+    ];
     this.setState({ attributes: newAttributes });
     this.props.onChange({
       attributes: newAttributes,
-      value: this.parseTags(Array.of(tags.value || tags.key)),
+      value: `${
+        this.props.value.value.length > 0 ? `${this.props.value.value},` : ''
```

Comment:
Please shorten this line by using destructurization:
`const { value } = this.props.value`